tests/test_units.py

```diff
@@ -0,0 +1,130 @@
+import pint
```

Comment:
Instead of using the pint unit registry, I would use the one you have in `units.py`, as demonstrated [here](https://pint.readthedocs.io/en/stable/getting/pint-in-your-projects.html)

Reply:
I'll take care of this for you.